mysql-test/suite/rocksdb/t/index_stats_basic-master.opt

```diff
@@ -0,0 +1,3 @@
+--skip-rocksdb_debug_optimizer_no_zero_cardinality
+--rocksdb_table_stats_sampling_pct=100
```

Comment:
I missed these in many previous pull requests, but, in any of the myrocks .opt files we need to prefix all myrocks options with --loose for the cases/platforms where myrocks isn't built but the default suites are run.  This isn't as much of an issue for us anymore but it does still exist.  @inikep we should add this info the MyRockks merge confluence doc.

Reply:
1. I updated the MyRocks merge doc
2. I will fix the issue in 5.7 and port it later to 8.0